docker-app/qfieldcloud/subscription/models.py

```diff
@@ -627,6 +656,45 @@ def active_users_count(self) -> int:
 
         return self.active_users.count()
 
+    @property
+    def organization_members_count(self) -> int:
+        # if non-organization account, then it is always 1 user
+        if not self.account.user.is_organization:
+            return 1
+
+        return OrganizationMember.objects.filter(
+            organization_id=self.account.user.pk
+        ).count()
+
+    @property
+    def included_storage_mb(self) -> int:
```

Comment:
I think we can kill `self.included_storage_mb` and use `self.included_storage_bytes` or at least reuse the logic there and just `self.included_storage_bytes() / 1000 / 1000`